.ci/before_install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -eux
+
+if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
```

Comment:
String comparison in bash is done using single 'equal '=' not double euqual.
Also please Use [[ and ]] instead of [ and ], because double square brackets will throw an error, if an undefined variable is used, single square bracket will just continue

Reply:
True to the "set -e". Nevertheless [[ and ]] is the standard format for a safer code.

I'd personally leave the comparison as is, since it is then self explaining